Libplanet/Net/Protocols/KademliaProtocol.cs

```diff
@@ -127,15 +127,6 @@ private readonly DifferentAppProtocolVersionEncountered
             }
             catch (Exception e)
             {
-                if (findPeerTasks.All(task =>
-                    task.IsFaulted &&
-                    !(task.Exception is null) &&
-                    task.Exception.InnerExceptions.All(ex => ex is TimeoutException)))
-                {
-                    throw new TimeoutException(
```

Comment:
Do we log inner exceptions in other places?  If not it might better to log them here. 🤔

Reply:
I thought the inner exceptions are logged here. https://github.com/planetarium/libplanet/pull/933/files#diff-ae527d5afcdf41da4a57b594cab3cfefL655